gluestick/singer.py

```diff
@@ -196,6 +205,75 @@ def deep_convert_datetimes(value):
         return value.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
     return value
 
+def parse_objs(x):
```

Comment:
How come you moved this here from `etl_utils.py`?

Reply:
to avoid circular imports, `to_export` here uses `to_singer` from `singer.py` and I wanted to use this function in `singer.py `functions :c